packages/lablqml/lablqml.0.6.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer:   "kakadu.hafanana@gmail.com"
+authors:      "kakadu.hafanana@gmail.com"
+homepage:     "http://kakadu.github.io/lablqt/"
+bug-reports:  "https://github.com/kakadu/lablqml/issues"
+dev-repo: "git+https://github.com/Kakadu/lablqml.git"
+tags: [ "gui" "ui" "qt" ]
+
+build: [
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"] {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]        {os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make demos"]  
+    { (os-distribution = "alpine"  | os-distribution = "centos" | os-distribution = "fedora") & with-test }
+  
+  ["./configure"]            { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
+  [make]                     { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
+  [make "demos"] { with-test & os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" } 
+]
+install: [
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make install"]       { os-distribution  = "alpine" | os-distribution  = "centos" | os-distribution  = "fedora" }
+  [make "install"]                                                                  { os-distribution != "alpine" & os-distribution != "centos" & os-distribution != "fedora" }
+]
+remove: [make "uninstall"]
```

Comment:
```suggestion
```
This is not necessary since opam 2.0

Reply:
Can you link a documentation page about this, please?

Reply:
https://opam.ocaml.org/doc/Manual.html#opamfield-remove